README.md

```diff
@@ -76,103 +88,156 @@ Of course, this Guide can’t replace professional advice. But talk to anyone wh
-    - **Incentive stock options (ISOs)** (also called statutory stock options)
-    	- ISOs are common for employees because they have the possibility of being more favorable from a tax point of view than NSOs. They can only be granted to employees (not independent contractors or directors who are not also employees). But ISOs have a number of limitations and conditions and can also create difficult tax consequences. More on this below.
-    - **Nonstatutory stock options (NSOs)** (also called non-qualifying stock options, or NQOs)
-- 🔹 Sometimes, to help you lower your tax burden, the company makes it possible to **early exercise** (or forward exercise) stock options. This means you exercise them even before they vest: you exercise them and you become a stockholder, but the company has the right to repurchase the *unvested* shares (at lower the price you paid or the fair market value of the shares) if you quit working for the company. The company will typically repurchase the unvested shares should you leave the company before the stock you’ve purchased vests.
+- 🔹 At some point early on, generally before the first employees are hired, stock will be allocated to a stock [**option pool**](http://www.investopedia.com/terms/o/option-pool.asp). A typical size for this is 20% of the stock of the company, but it can be 10%, 15%, or other sizes. Once the pool is established, then the company’s board of directors grants pieces of it to employees as they join the company. Often, the whole pool is never used. The size of the pool is not just about how generous the company is with employees; it is determined by [complex factors between founders and investors](http://venturehacks.com/articles/option-pool-shuffle).
+- 🔹☝️ Compensatory stock options come in two flavors, **incentive stock options (or ISOs)** and **non-qualifying stock options (or NQOs)**. Confusingly, lawyers can’t even agree on the names of these two kinds of stock options, so you’ll also see them called **statutory stock options** and **non-statutory stock options (or NSOs)**, respectively.
+	- ISOs are common for employees because they have the possibility of being more favorable from a tax point of view than NSOs (below). They can only be granted to employees (not independent contractors or directors who are not also employees). But ISOs have a number of limitations and conditions and can also create difficult tax consequences. More on this below.
+- 🔹 Sometimes, to help you lower your tax burden, the company makes it possible to **early exercise (or forward exercise)** stock options. This means you exercise them even before they vest, which means you become a stockholder. However, the company has the right to repurchase the *unvested* shares, at a lower price than what you paid, or the fair market value of the shares, if you quit working for the company. The company will typically repurchase the unvested shares should you leave the company before the stock you’ve purchased vests.
```

Comment:
Let's directly tie how early exercise and lowering your tax burden are related. Early exercise allows you to buy your shares at the current FMV of the stock, with an 83(b), you lock in tax treatment on that price vs. waiting to exercise, buying at a (much) higher price, and getting slapped with a tax bill big enough to make your stomach upside down and inside out.

Reply:
🚧

Reply:
🚧